cmake/ProcessOptions.cmake

```diff
@@ -594,6 +596,7 @@ function( NEST_PROCESS_WITH_BOOST )
       set( BOOST_LIBRARIES "${Boost_LIBRARIES}" PARENT_SCOPE )
       set( BOOST_INCLUDE_DIR "${Boost_INCLUDE_DIR}" PARENT_SCOPE )
       set( BOOST_VERSION "${Boost_MAJOR_VERSION}.${Boost_MINOR_VERSION}.${Boost_SUBMINOR_VERSION}" PARENT_SCOPE )
+      include_directories( ${Boost_INCLUDE_DIRS} )
```

Comment:
Both `Boost_INCLUDE_DIR` and `Boost_INCLUDE_DIRS` will probably point to the same location. However, based on the examples in [the documentation](https://cmake.org/cmake/help/latest/module/FindBoost.html), and [this answer](https://discourse.cmake.org/t/findboost-difference-between-boost-include-dir-and-boost-include-dirs/855/2) in the CMake forum, `Boost_INCLUDE_DIRS` is the correct variable to use.

Reply:
I have just updated PR #1477 where I used,
`include_directories( ${Boost_INCLUDE_DIR} )`

should I change to
`include_directories( ${Boost_INCLUDE_DIRS} )`
and use
`set( BOOST_INCLUDE_DIR "${Boost_INCLUDE_DIRS}" PARENT_SCOPE )`
as commented by @hakonsbm ?

And just to add, BOOST_INCLUDE_DIR is a variable set on the scope above the current (PARENT_SCOPE, [see manual](https://cmake.org/cmake/help/latest/command/set.html)), and the following code would not work as expected, as it will contain the previous value of tha variable.
`include_directories( ${BOOST_INCLUDE_DIR} )`